clustering/ee/spi/src/main/java/org/wildfly/clustering/ee/expiration/ExpirationMetaData.java

```diff
@@ -37,7 +37,7 @@ public interface ExpirationMetaData extends Expiration {
     default boolean isExpired() {
         if (this.isImmortal()) return false;
         Instant lastAccessedTime = this.getLastAccessTime();
-        return (lastAccessedTime != null) ? lastAccessedTime.plus(this.getTimeout()).isBefore(Instant.now()) : false;
+        return (lastAccessedTime != null) ? !lastAccessedTime.plus(this.getTimeout()).isAfter(Instant.now()) : false;
```

Comment:
I might be misunderstanding something but won't this still return false if lastAccessedTime+timeout == now?

Reply:
I don't think so.
Current behavior is:
```java
instant.isBefore(now)
```
i.e. instant < now.  However, we want this to be inclusive, such that instant <= now

```java
instant.isAfter(now)
```
means instant > now, thus the inverse:
```java
!instant.isAfter(now)
```
means instant <= now, which is what we want.

Reply:
👍 It's the subtle added '!' in the diff that's key.